packages/react/src/components/DatePicker/DatePicker.js

```diff
@@ -210,6 +242,8 @@ export default class DatePicker extends Component {
       'uk', // Ukrainian
       'vn', // Vietnamese
       'zh', // Mandarin
+      'zh-cn', // Mandarin alternate
```

Comment:
The list here should be in sync with https://github.com/flatpickr/flatpickr/tree/v4.6.1/src/l10n, but `zh-cn` doesn't seem to be there.

Reply:
Hmm, yeah this is a product of i18n spitting out zh-cn. I end up re-casting it to `zh` and `zh-tw` to `zh_tw`. We can do that locally, however I felt like enough people might just be passing through that it would be worth the it.